### PR TITLE
Ncnews 5b

### PR DESCRIPTION
**Pull request by ckloudy**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -64,3 +64,37 @@ describe('/api/articles/99(404) & /api/articles/notAnId(400)', () => {
...
+			return request(app).patch('/api/articles/1').send(inc_vote).expect(201).then((response) => {
+				expect(response.body.article.article_id).toBe(1);
+				expect(response.body.article.votes).toBe(95);
+			});
+		});
+	});
+});
+
+describe('/api/articles/99(404) & /api/articles/notAnId(400)', () => {
+	describe('PATCH ERROR', () => {
```
Comment (k-huggs) [inline __tests__/app.test.js:88]: add 400 error for {inc_votes: banana}

File: models/articles.model.js [hunk 2]
```diff
@@ -13,3 +13,15 @@ exports.selectArticleById = (id) => {
 			return rows[0];
 		});
 };
+
+exports.updatedVotesByArticleById = (id, update) => {
+	return db
+		.query(`UPDATE articles SET votes = votes + $2 WHERE article_id = $1 RETURNING*;`, [ id, update ])
+		.then(({ rows }) => {
+			console.log(rows);
```
Comment (k-huggs) [inline models/articles.model.js]: delete clg

**Comment by k-huggs** [comment]
add final 400 error test then merge

File: __tests__/app.test.js (hunk 1 above)
Comment (ckloudy) [reply __tests__/app.test.js:88]: This test has been added

File: models/articles.model.js (hunk 2 above)
Comment (ckloudy) [reply models/articles.model.js]: clg removed

**Pull request merged**